script/build.js

```diff
@@ -116,6 +116,13 @@ smith.destination(`../build/${options.buildtype}`);
 // }
 // smith.use(ignore(ignoreList));
 
+const ignore = require('metalsmith-ignore');
+const ignoreList = [];
+if (options.buildtype === 'production') {
+  ignoreList.push('gi/*');
```

Comment:
I think this uses the relative path from the `content` folder, so it should probably be `'gi-bill-comparison-tool/*`.